yarn-project/p2p/src/testbench/testbench.ts

```diff
@@ -132,7 +134,9 @@ async function main() {
     logger.info('Workers Ready');
 
     // Send tx from client 0
-    const tx = await Tx.random(/* randomProof */ true);
+    const tx = await mockTx(1, {
```

Comment:
reverted back to using mock tx instead of random to pass validators in testbench